packages/beauty-portal-theme/src/components/Chevron/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react';
+
+interface ChevronInterface {
+  className: string;
+  height: number;
+  width: number;
+  fill: string;
+}
+
+const Chevron: FunctionComponent<ChevronInterface> = chevron => {
+  return (
+    <svg
+      className={chevron.className}
```

Comment:
may need one of the aria attributes for accessibility

Reply:
Added as per the other SVG files in the project.